### PR TITLE
fix: (kubernetes source) Change `timestamp_key` back to `message_key`

### PR DESCRIPTION
**Pull request by ktff**
Field key in [this line](https://github.com/timberio/vector/blob/df8e103b937cc86d9ef12cf6b2638a4003f75b47/src/sources/kubernetes/mod.rs#L124) was accidentally changed in #1769 from message to timestamp. 

This was causing messages to retain newline at their ends. 

This was also causing kubernetes tests to fail.

PR changes this key back to message.

### CODE REVIEW

**Comment by Hoverbear** [comment]
Oh no, I'm so sorry. :(

File: src/sources/kubernetes/test.rs [hunk 1]
```diff
@@ -112,7 +112,7 @@ spec:
         emptyDir: {}
       containers:
       - name: vector
-        image: ktff/vector-kube-watch-client:latest
+        image: ktff/vector-kube-newline-bug:latest
```
Comment (Hoverbear) [inline src/sources/kubernetes/test.rs]: Is this intentional?

**Review by Hoverbear — APPROVED** [review]
Rest looks good, sorry about that!

**Comment by binarylogic** [comment]
@ktff feel free to merge when you’re ready. I’d like to get this into 0.8.0.

cc/ @Jeffail

**Comment by binarylogic** [comment]
Also, is there any reason rests didn’t catch this?

**Comment by ktff** [comment]
@binarylogic 

> Also, is there any reason tests didn’t catch this?

They did, that's how I noticed it while testing #1888 . But the kubernetes tests are currently disabled on CL.

File: src/sources/kubernetes/test.rs (hunk 1 above)
Comment (ktff) [reply src/sources/kubernetes/test.rs]: Yes. [This](https://github.com/timberio/vector/issues/260#issuecomment-565836899) note explains why.

**Pull request merged**